### PR TITLE
Feature #162058565 - HCA landing page improvements

### PR DESCRIPTION
**Pull request by monicajianu**
- Created an additional base JSP which doesn't contain any EBI branding
- Created special HCA footer containing EBI, Sanger and Wellcome logos
- Added boxes with Tools, People and Data to the HCA landing page
- Updated `atlas-homepage-cards` depedency to latest version

### CODE REVIEW

File: sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    <p> Single cell Expression Atlas provides access to the analysis results of
+        <fmt:formatNumber pattern="#,##0" value="${hcaExperimentsCount}"/> data sets from the Human Cell Atlas
+        and <fmt:formatNumber pattern="#,##0" value="${humanExperimentsCount}"/> data sets in <i>Homo sapiens</i>
+        from other studies.
+    </p>
+    <p>
+        All studies have been re-analysed using our standardised pipeline
+        (<a href="https://github.com/nunofonseca/irap">iRAP</a>). Pipelines and web services are
+        being developed by the Gene Expression Team at EMBL-EBI, funded by the
```
Comment (alfonsomunozpomer) [inline sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp]: Judging from https://wellcome.ac.uk/about-us, it looks like that under the new Wellcome Trust branding “the” isn’t used:
image

**Review by alfonsomunozpomer — APPROVED** [review]
Since the wording isn’t final, all good. You might want to remove an article to increase brand engagement.

File: sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp (hunk 1 above)
Comment (monicajianu) [reply sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp]: Yeah, I did take the liberty to remove "Trust" from the original version, but I wasn't sure how you're meant to use the new name 😄 You're right, I should remove the "the" as well...

Comment (alfonsomunozpomer) [reply sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp]: Now Wellcome is like your new best friend: “Here’s Wellcome! Say hi to all our users!”.

Comment (monicajianu) [reply sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page/data.jsp]: "Welcome, Wellcome!"

**Pull request merged**